tests/test_edgeql_expressions.py

```diff
@@ -380,6 +397,11 @@ async def test_edgeql_expr_literals_02(self):
                     r'''SELECT <int64>'3689348814741900000000000' ''',
                 )
 
+        with self.assertRaisesRegex(edgedb.EdgeQLSyntaxError,
+                                    'expected digit after dot'):
```

Comment:
These sort of tests would probably be better placed in `test_edgeql_syntax`.

Reply:
+1. Better keep all syntax tests consolidated in one place.

Reply:
I've move tests from edgeql_expressions. But left other ones in ddl. Or should I move DDL tests too?

Reply:
If the purpose of those tests is to test uninterpreted handling of literals without necessarily hitting Postgres, then yes.

Reply:
Done.